src/commonMain/kotlin/com/github/doyaaaaaken/kotlincsv/dsl/context/CsvReaderContext.kt

```diff
@@ -119,6 +124,7 @@ enum class ExcessFieldsRowBehaviour {
  */
 @CsvDslMarker
 class CsvReaderContext : ICsvReaderContext {
+    override var logger: Logger = LoggerNop
```

Comment:
Please mention this on README page.
https://github.com/doyaaaaaken/kotlin-csv#customize

Reply:
Done :)